Algebroids/gap/FpCategories.gi

```diff
@@ -2356,15 +2374,15 @@ InstallMethod( SieveFunctor,
         [ IsFpCategory ],
         
   function ( B )
-    local Bop, H, Sieves;
+    local Bop, sFinSets, Sieves;
     
     Bop := OppositeFpCategory( B );
     
-    H := RangeCategoryOfHomomorphismStructure( B );
+    sFinSets := RangeCategoryOfHomomorphismStructure( B );
```

Comment:
Here and in the following two functions there is no assert after `sFinSets := ...;`. Is this deliberate?

Reply:
These are user-level functions that immediately call other functions that have the assertion.

Reply:
See https://github.com/homalg-project/CategoricalTowers/pull/227 for a suggested improvement.